docs/UserGuide.adoc

```diff
@@ -400,29 +399,51 @@ To match all the patients in ORGANice, do the following:
 image::Patient Matches Displayed.png[width=790]
 +
 
+Following this, you can sort the matches based on different categories to deduce which patient to prioritize.
++
+Refer to refer to <<Sorting matches: `sort`>> for more information.
 
 === Sorting matches: `sort`
-Users can specify how they want the patient-donor pairs to be sorted. +
-Sorting is done by passing in optional parameters to the `match` command. +
+This section specifies how a list of matches can be sorted to determine which patient to prioritize for organ transplant.
 
-Matches can be sorted according to three categories:
+**The sort command can only be used after a match command** to sort the resulting list of patient-donor pairs generated.
+The following specifies three categories by which you can sort the matches.
```

Comment:
I expect 3 bullet points if you said this, so can add in the 3 categories as bullet points her maybe

Reply:
Added.